src/index.js

```diff
@@ -1 +1,21 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { getScores, addScore } from './modules/scoresHandle.js';
+import { displayScores } from './modules/createList.js';
+
+const submitBtn = document.querySelector('.btn-submit');
+const refreshBtn = document.querySelector('.btn-refresh');
+
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  addScore();
```

Comment:
- [ ] [OPTIONAL] For a good user experience, you may consider emptying the input fields when a score is submitted.